lib/readline.js

```diff
@@ -992,7 +999,7 @@ function emitKeypressEvents(stream, iface) {
   stream[ESCAPE_DECODER] = emitKeys(stream);
   stream[ESCAPE_DECODER].next();
 
-  const escapeCodeTimeout = () => stream[ESCAPE_DECODER].next('');
+  const escapeCodeTimeoutFn = () => stream[ESCAPE_DECODER].next('');
```

Comment:
And if you don't introduce a new intermediate variable, you don't need to rename this.